src/Compiler/Microsoft.AspNetCore.Razor.Language/src/RazorCodeGenerationOptions.cs

```diff
@@ -146,4 +148,9 @@ public static RazorCodeGenerationOptions CreateDesignTime(Action<RazorCodeGenera
     /// Gets a value that determines if enhanced line pragmas are to be utilized.
     /// </summary>
     public virtual bool UseEnhancedLinePragma { get; }
+
+    /// <summary>
+    /// Gets a value that suppresses unique ids for testing purposes.
+    /// </summary>
+    internal string SuppressUniqueIds { get; private protected init; }
```

Comment:
So this property's content is used instead of the unique ID? Perhaps the doc comment could say that. E.g., "Value used instead of unique IDs for testing purposes or `null` to use unique IDs."